lib/ash_authentication/strategies/password.ex

```diff
@@ -94,7 +94,8 @@ defmodule AshAuthentication.Strategy.Password do
   See the [Testing guide](/documentation/topics/testing.md) for tips on testing resources using this strategy.
   """
 
-  defstruct confirmation_required?: false,
+  defstruct require_confirmed_with?: false,
+            confirmation_required?: false,
```

Comment:
This is no longer required.

Reply:
Sorry, just to be sure I understand, you are suggesting that the explicit use of `confirmation_required?` is replaced by `require_confirmed_with` ? Isn't this going to be a breaking change for whatever user code is using the flag ?

Reply:
Also, the `confirmation_required?` relates to the will to have a `password_confirmation` **field**, if I get this correctly (at least that's what I read in the `Password.md` file which was going to update now.

Reply:
Oh sorry. It was a brain fart on my end. That option is whether or not password confirmation is required 😂 ignore me.

Reply:
Oh, happy I spotted it, I'd have been changing this regardless :P